README.md

```diff
@@ -13,51 +28,36 @@ Getting started
 1. If you are using the `develop` branch of vector, you will probably need to
    rebuild one of the dependencies, due to https://github.com/npm/npm/issues/3055:
    `(cd node_modules/matrix-react-sdk && npm install)`
-1. Start the development builder and a testing server: `npm start`
-1. Wait a few seconds for the initial build to finish (the command won't
-   terminate: it's running a web server for you).
-1. Open http://127.0.0.1:8080/ in your browser to see your newly built Vector.
+1. Configure the app by copying `config.sample.json` to `config.json` and modifying
```

Comment:
Oh, actually I did mean config.sample.json but I see what you mean: if you made a config.json before packaging, would you expect it to be in the package? Relatedly, I now can't easily un-tar a new version of vector onto my server and update the symlink as I have to copy the config.json over (unless I untar over the old directory but then I'll get stale files). I can't find anyone who's solved this nicely.

I guess excluding the config file from the tarball prevents us from accidentally shipping one we didn't intend to package, and doesn't really hurt. Probably best done in a different PR though rather than shoehorning it into here.